tests/helpers/collection-reactivity.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```

Comment:
we may improve this moving forward.
For now, as I'm gonna work on monorepo and v2 addon format conversion, I will need to make this change to not pin `@typescript-eslint/eslint-plugin` and keep changelist focused